deploy/kubernetes/rbac.yaml

```diff
@@ -50,6 +50,9 @@ rules:
   - apiGroups: [""]
     resources: ["nodes"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["volumeattachments"]
+    verbs: ["list"]
```

Comment:
Should there be also `"watch"`? I hope `vaLister.List()` below does not list all VolumeAttachments from the API server.

Reply:
Ah you're right, we only need to watch on the API server and list from the cache. However i'm wondering do we still need a List/Get to populate the cache?

Reply:
`list` is needed to populate the cache, I am not sure about `get`, IMO it won't harm to have it in the RBAC - all the other objects have it.

Reply:
done!